.claude/docs/lh/performance-analysis.md

```
@@ -0,0 +1,228 @@
+|------|--------|------|-------------|------|
+| **FCP** (First Contentful Paint) | 1.1s | 0.99 | ≤ 1.8s | 🟢 좋음 |
+| **LCP** (Largest Contentful Paint) | 5.6s | 0.18 | ≤ 2.5s | 🔴 매우 나쁨 |
+| **TBT** (Total Blocking Time) | 1,810ms | 0.09 | ≤ 200ms | 🔴 매우 나쁨 |
+| **CLS** (Cumulative Layout Shift) | 0 | 1.00 | ≤ 0.1 | 🟢 완벽 |
+| **SI** (Speed Index) | 4.7s | 0.68 | ≤ 3.4s | 🟡 보통 |
+| **TTI** (Time to Interactive) | 5.8s | 0.67 | ≤ 3.8s | 🟡 보통 |
+| **TTFB** (서버 응답) | 0ms | 1.00 | ≤ 800ms | 🟢 완벽 |
+
+### Lighthouse 점수 계산 가중치
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)